EncounterModel.m

```diff
@@ -0,0 +1,357 @@
+classdef EncounterModel < handle
```

Comment:
Is this file a copy of `code\matlab\@EncounterModel\EncounterModel`? If a copy, please remove this duplicate. If this code was modified, can you please update of the existing file?

Reply:
This file is not a duplicate; there is a minor modification Line 180 
           if contains((parameters_filename), 'rotorcraft')||contains(lower(parameters_filename), 'helicopter')".......
This line was added to enable setting minimal allowable speed to 0 for Canadian Helicopter model.

Reply:
This code was modified.

Reply:
However, I just realized that there are duplicates all over the main folder! This was corrected